server/util/clickhouse/schema/schema.go

```diff
@@ -125,11 +127,16 @@ func (i *Invocation) ExcludedFields() []string {
 		"RedactionFlags",
 		"CreatedWithCapabilities",
 		"Perms",
+		"ParentInvocationID",
 	}
 }
 
 func (i *Invocation) AdditionalFields() []string {
-	return []string{}
+	return []string{
+		// Invocation IDs are stored as UUIDs (without dashes), so store the parent invocation ID
+		// as a UUID, so it can be easily joined.
+		"ParentInvocationUUID",
+	}
 }
```

Comment:
why is ParentInvocationUUID is in this list but not InvocationUUID? (sorry I'm not super familiar with the clickhouse schema)

Reply:
We save InvocationUUID explicitly in the mysql db (https://github.com/buildbuddy-io/buildbuddy/blob/07185758c263aae0210fc63b4149af8172f698f6/server/tables/tables.go#L1037). @luluz66 there aren't any reasons why I'd have to store my new UUID in mysql right? We can just convert it to UUID before storing in click house?